RadarSDK/RadarAPIClient.m

```diff
@@ -1101,14 +1101,23 @@ - (void)validateAddress:(RadarAddress *)address completionHandler:(RadarValidate
                     }];
 }
 
-- (void)geocodeAddress:(NSString *)query completionHandler:(RadarGeocodeAPICompletionHandler)completionHandler {
+- (void)geocodeAddress:(NSString *)query 
+                layers:(NSString *)layers
+               country:(NSString *)country
```

Comment:
These should be `NSArray`'s that are nullable, like: https://github.com/radarlabs/radar-sdk-ios/blob/b8311225e9a2728ea6205db45b844b1704768eb1/RadarSDK/Radar.m#L767